lepton/gcp.go

```diff
@@ -681,3 +586,177 @@ func createArchive(archive string, files []string) error {
 func (p *GCloud) ResizeImage(ctx *Context, imagename string, hbytes string) error {
 	return fmt.Errorf("Operation not supported")
 }
+
+// CreateVolume creates local volume and converts it to GCP format before orchestrating the necessary upload procedures
+// TODO treat GCS object and Compute Image as temporary and delete after volume is created?
```

Comment:
yes, please delete the tmp. disk image and use the volume as the new artifact - if one wants to clone it or version it or whatever they should use the volume as the end artifact to do so

Reply:
alright will update this